File: pgm/ABC_Java/ABC_307_A.java
import java.io.*;
// import java.math.*;
import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final int N = Integer.parseInt (reader.readLine());
        final int[] A = Arrays.stream (reader.readLine().split (" "))
                            .mapToInt (Integer::parseInt)
                            .toArray();
        final int zero = 0;
        final int seven = 7;
        final String sp = " ";
        StringBuilder sb = new StringBuilder();
        String s = "";
        int sum = 0;
        int j = 0;
        int next = 0;
        for (int i = 0; i < N; i++) {
            next = j + seven;
            for (; j < next; j++) {
                sum += A[j];
            }
            sb.append (s);
            sb.append (String.valueOf (sum));
            s = sp;
            sum = zero;
        }
        System.out.println (sb.toString());
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_307_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.StringTokenizer

// import kotlin.math.*

const val blank: String = ""
const val space: String = " "
const val seven: Int = 7
const val zero: Int = 0

fun main() {
    val N: Int = readln().toInt()
    val A: IntArray = readln().split(space).map { it.toInt() }.toIntArray()
    var s: String = blank
    var sum: Int = zero
    var j: Int = zero
    repeat(N) {
        val next = j + seven
        while (j < next) {
            sum += A[j]
            j++
        }
        print(s)
        print(sum)
        s = space
        sum = zero
    }
    println()
    kotlin.system.exitProcess(0)
}